test/propagated-properties.html

```diff
@@ -37,7 +37,9 @@
         initial-position="1980-01-01"
         show-week-numbers
         label="Birth date and time"
+        error-message="error-message"
         disabled
+        required
```

Comment:
Tests should be added for these also in the ``describe(`Initial property values`` block.

Reply:
Added.